backend/sozisel/lib/sozisel/model/quizzes/quiz.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Sozisel.Model.Quizzes.Quiz do
+  use Sozisel.Model.Schema
+  import Ecto.Changeset
+
+  alias Sozisel.Model.Quizzes.QuizQuestion
+
+  @type t :: %__MODULE__{
+          duration_time: Integer.t(),
+          number_of_targets: Integer.t(),
+          tracking_mode: Boolean.t(),
+          quiz_questions: [QuizQuestion]
+        }
+
+  @primary_key false
+
+  embedded_schema do
+    field :duration_time, :integer
+    field :number_of_targets, :integer
```

Comment:
I don't like this options. I can just imagine that it means how many people should get that event but it would strongly vary depending on number of people in the room.